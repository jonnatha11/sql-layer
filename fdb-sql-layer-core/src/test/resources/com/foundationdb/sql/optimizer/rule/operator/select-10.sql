SELECT customers.name, order_date, child.name FROM customers,orders,parent,child
 WHERE customers.cid = orders.cid
   AND parent.id = child.pid
   AND customers.name = parent.name
   AND order_date > '2011-01-01'
   AND child.name <> 'Smith'
