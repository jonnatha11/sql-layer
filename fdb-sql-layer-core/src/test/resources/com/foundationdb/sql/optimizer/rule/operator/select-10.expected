PhysicalSelect@53ecb4e2[name, order_date, name]
  Map_NestedLoops(0)
    Flatten_HKeyOrdered(test.customers INNER test.orders)
      GroupLookup_Default(Index(test.orders.order_date) -> test.customers, test.orders)
        IndexScan_Default(Index(test.orders.order_date), order_date > DATE '2011-01-01' ASC, cid, oid)
    Project_Default(Bound(0, 1), Bound(0, 4), Field(2))
      Select_HKeyOrdered(Field(2) != 'Smith')
        Filter_Default(test.child)
          GroupLookup_Default(Index(test.parent.name) -> test.child)
            IndexScan_Default(Index(test.parent.name), name = Bound(0, 1), id ASC)